packages/valory/agents/test_libp2p/aea-config.yaml

```diff
@@ -1,49 +0,0 @@
-agent_name: test_libp2p
```

Comment:
I removed the agent since putting package tests in an agent package does not work as intended. There were so many errors caused by the loader not being able to load all packages necessary. I tried updating the test command to fix this but it's a lot for a single PR. So I moved the tests from the agent package to a dummy connection package.